File: 03 Podminky a cykly/02 fizzbuzz.ts
function fizzbuzz(n: number) {
  // and = &&, or = ||, not = !
  if (n % 3 === 0 && n % 5 === 0) return `fizzbuzz`;
  if (n % 3 === 0) return `fizz`;
  if (n % 5 === 0) return `buzz`;
  return n;
}

for (let i = 0; i <= 20; i++) console.log(fizzbuzz(i));

let i = 0;
while (i <= 20) {
  console.log(fizzbuzz(i));
  i++;
}

i = 0;
do {
  console.log(fizzbuzz(i));
  i++;
} while (i <= 20);

File: 03 Podminky a cykly/02 fizzbuzz.py
def fizzbuzz(n):
    if n % 3 == 0 and n % 5 == 0:
        return "fizzbuzz"
    if n % 3 == 0:
        return "fizz"
    if n % 5 == 0:
        return "buzz"
    return n


for i in range(21):
    print(fizzbuzz(i))

i = 0
while i <= 20:
    print(fizzbuzz(i))
    i += 1
